app/static/playlist.js

```diff
@@ -312,32 +313,63 @@ export default class PlaylistLabelRenderer {
     return str.length > max ? `${str.substr(0, max - 3)}...` : str;
   }
 
-  handleTapMessage() {
-    // UPDATE 'COLLECTED' UI
-    if (!this.state.isAnimatingCollect) {
-      // Debounced with isAnimatingCollect
-      this.state.isAnimatingCollect = true;
-
-      // Animation plays: collect -> hidden -> collected -> hidden -> collect
-      const collectElement = document.getElementById("collect");
-      const { collectClassname } = this.state;
-      collectElement.className = `${collectClassname} hidden`;
-      window.setTimeout(function timeout1() {
-        collectElement.innerHTML = "COLLECTED";
-        collectElement.className = `${collectClassname} active`;
-      }, 500);
-      window.setTimeout(function timeout2() {
-        collectElement.className = `${collectClassname} active hidden`;
-      }, 3000);
-      window.setTimeout(
-        function timeout3() {
-          collectElement.className = collectClassname;
-          collectElement.innerHTML = "COLLECT";
-          this.state.isAnimatingCollect = false;
-        }.bind(this),
-        3500
+  openErrorDialogue(errorText) {
+    const errorDialogueElement = document.getElementById("error-dialogue");
+    errorDialogueElement.className = "error-dialogue open";
+    const errorDialogueTextElement = document.getElementById(
+      "error-dialogue-text"
+    );
+    errorDialogueTextElement.innerHTML = errorText;
+    window.clearTimeout(this.state.errorDialogueCloseTimeout);
+    this.state.errorDialogueCloseTimeout = window.setTimeout(
+      this.closeErrorDialogue.bind(this),
+      3000
+    );
+  }
+
+  closeErrorDialogue() {
```

Comment:
What was your reasoning for using css to close the dialog instead of javascript as used in the Interactive Label? https://github.com/ACMILabs/interactive-label/blob/main/app/static/app.js#L384

I don't have a strong opinion over which we use, but maybe we should pick one and stick with it?

Reply:
Yeah I'm down with that - and given we've already put through the Interactive Label I'll switch this to js.